fastlane/lanes/release.rb

```diff
@@ -363,7 +363,7 @@
     UI.important "Publishing release #{version_number} on GitHub"
 
     publish_github_release(
-      repository: GITHUB_REPO,
+      repository: GHHELPER_REPO,
```

Comment:
From reviewing https://github.com/wordpress-mobile/WordPress-Android/pull/21636, this change appears incorrect.